script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -3068,6 +3078,7 @@ There's a silent edge behind that smile that rapidly dispels my drowsiness.
 [sha:777a2aeefb9aa557508910165dd35487c7c506f6]{
 -- Page 187, Offset 42517.
 -- 「体も死徒に成りかけのまま戻らないようですし。
+// Make this a bit less lomg in en?
```

Comment:
> "It looks like your body is still close to that of a Dead Apostle, too.

Reply:
I think the 'not reverting' is the important part here. 

> It doesn't seem like you'll return to being human, either.

Reply:
This shouldn't be too hard to keep fairly close to the original Japanese without over simplifying and still keeping short.
>Your half-vampire body seems unlikely to return to normal, too.